prison-core/src/main/resources/lang/mines/de_DE.properties

```diff
@@ -27,3 +27,4 @@ block_not_removed=&7Dieser Block ist nicht in dieser Mine vorhanden.
 block_deleted=&7Block &3%1 &7von der Mine entfernt &3%2&7.
 mine_redefined=&7Erfolgreich die Mine &3neudefiniert &7.
 missing_world=&7Die Welt, in der die Mine erstellt wurde, kann nicht gefunden werden.
+block_search_blank=&7Enter a value to search for a block.&7.
```

Comment:
Remove this, the localization system defaults to the english variant anyway.

Reply:
Just fixed these and pushed the commit to bleeding, so should be there by now.